src/test/java/org/folio/rest/impl/TemplateRequestTest.java

```diff
@@ -96,7 +97,6 @@ public static void setUpClass(final TestContext context) throws Exception {
         PostgresClient.setConfigFilePath(postgresConfigPath);
         break;
       case "embedded":
-        PostgresClient.setIsEmbedded(true);
```

Comment:
All code that configures and starts and stops the database for testing should be removed. PostgresClient was updated to do this automatically: https://github.com/folio-org/raml-module-builder#postgresql-integration

Reply:
The database configuration in TemplateRequestTest.java is different from that in TemplateDeletionTest.java and RestVerticleTest.java and should be changed, for example by using the configuration provided by PostgresClient: https://github.com/folio-org/raml-module-builder#postgresql-integration